packages/styling/icons/new-window.svg

```diff
@@ -0,0 +1,3 @@
+<svg id="new-window" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```

Comment:
Moet dat icoon echt 2maal in de codebase worden opgenomen?

Reply:
1x in het theme, 1x in de styling